extensions/wikia/WikiFactory/DeadWikis/maintenance.php

```diff
@@ -292,10 +290,17 @@ protected function getWikisList() {
 			"l.city_created < \"".wfTimestamp(TS_DB,strtotime(self::$FETCH_TIME_LIMIT))."\"",
 			"(v.city_flags IS NULL OR v.city_flags & " . WikisModel::FLAG_OFFICIAL . " = 0)",
 		);
-		if (!is_null($this->from))
-			$where[] = "l.city_id >= ".intval($this->from);
-		if (!is_null($this->to))
-			$where[] = "l.city_id <= ".intval($this->to);
+		if (!is_null($this->ids) && !empty($this->ids)) {
```

Comment:
You only need to check `empty()` here.  The other lines look at `is_null` presumably to allow zero as a valid ID (though while a valid integer, its not a valid city ID).  If `$this->ids` has never been set, `empty()` still works.  Also, when there's a full `if/else` I always suggest simplifying and arranging them so that the if conditional doesn't have to be negated, e.g.:
```php
if (empty($this->ids)) {
    // Do stuff with $this->from and $this->to
} else {
    // Do stuff with $this->ids
}
```